commands/checkout.go

```diff
@@ -74,6 +82,11 @@ func transformCheckoutArgs(args *Args) error {
 		return err
 	}
 
+	if idx := args.IndexOfParam("--no-branch"); idx >= 0 {
+		args.RemoveParam(idx)
+		flagCheckoutNoBranch = true
```

Comment:
That's what I meant above ... for some reason it isn't getting set, but it seems to work in other commands. I must be missing something...